pages/resources/message.mdx

```diff
@@ -449,24 +449,37 @@ For the `attachments` array in Message Create/Edit requests, only the `id` is re
-| id                     | snowflake | The attachment ID                                                       |
-| filename               | string    | The name of file attached                                               |
-| uploaded_filename? ^2^ | string    | The name of the file pre-uploaded to Discord's GCP bucket               |
-| description?           | string    | The description for the file (max 1024 characters)                      |
-| content_type?          | string    | The attachment's [media type](https://en.wikipedia.org/wiki/Media_type) |
-| size                   | integer   | The size of file in bytes                                               |
-| url                    | string    | Source URL of the file                                                  |
-| proxy_url              | string    | A proxied url of the file                                               |
-| height?                | ?integer  | Height of file (if image)                                               |
-| width?                 | ?integer  | Width of file (if image)                                                |
-| ephemeral? ^1^         | boolean   | whether this attachment is ephemeral                                    |
+| Field                  | Type      | Description                                                                             |
+| ---------------------- | --------- | --------------------------------------------------------------------------------------- |
+| id                     | snowflake | The attachment ID                                                                       |
+| filename               | string    | The name of file attached                                                               |
+| uploaded_filename? ^2^ | string    | The name of the file pre-uploaded to Discord's GCP bucket                               |
+| description?           | string    | The description for the file (max 1024 characters)                                      |
+| content_type?          | string    | The attachment's [media type](https://en.wikipedia.org/wiki/Media_type)                 |
+| size                   | integer   | The size of file in bytes                                                               |
+| url                    | string    | Source URL of the file                                                                  |
+| proxy_url              | string    | A proxied url of the file                                                               |
+| height?                | ?integer  | Height of file (if image)                                                               |
+| width?                 | ?integer  | Width of file (if image)                                                                |
+| ephemeral? ^1^         | boolean   | whether this attachment is ephemeral                                                    |
```

Comment:
```suggestion
| ephemeral? ^1^         | boolean   | Whether this attachment is ephemeral                                                    |
```

While you're modifying this section why not update this for consistency

Reply:
Done